Src/AutoFakeItEasy2.UnitTest/AutoFakeItEasyCustomizationTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FakeItEasy;
+using Ploeh.AutoFixture.Kernel;
+using Xunit;
+
+namespace Ploeh.AutoFixture.AutoFakeItEasy.UnitTest
```

Comment:
We use renamed namespaces in [other places](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture.xUnit.net2/AutoDataAttribute.cs#L9). This makes it explicit what's going on. It may imply a bit more conversion work, but prevents subtle bugs where you think you're using one library, but in reality, it's not what you thought...